src/test/java/org/folio/bulkops/repository/RemoteFileSystemRepositoryTest.java

```diff
@@ -0,0 +1,86 @@
+package org.folio.bulkops.repository;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+import org.folio.bulkops.config.RepositoryConfig;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+
+import java.time.Duration;
+
+import static java.lang.String.format;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(classes = RemoteFileSystemRepository.class)
+@ContextConfiguration(classes = {RepositoryConfig.class})
+class RemoteFileSystemRepositoryTest {
+
+  private static final String S3_ACCESS_KEY = "minio-access-key";
+  private static final String S3_SECRET_KEY = "minio-secret-key";
+  private static final int S3_PORT = 9000;
+  private static String minio_endpoint;
+
+  private static final String INITIAL_FILE = "initial.txt";
```

Comment:
According to https://github.com/folio-org/mod-bulk-operations/pull/9/files#r1041922176 we don't need to operate with files on local fs. Please, use InputStreams instead of files.

Reply:
Here initial.txt is used as an argument of RemoteFileSystemRepository:put and this method does not accept InputStream

Reply:
This constant is used just to avoid duplication of test file names.